gtsfm/run_scene_optimizer.py

```diff
@@ -16,28 +17,32 @@
 TEST_ROOT = Path(__file__).resolve().parent.parent / "tests"
 
 
-@hydra.main(config_name="config")
-def run_scene_optimizer(cfg: DictConfig) -> None:
-    scene_optimizer: SceneOptimizer = instantiate(cfg.SceneOptimizer)
 
-    loader = FolderLoader(os.path.join(TEST_ROOT, "data", "set1_lund_door"), image_extension="JPG")
+def run_scene_optimizer() -> None:
+    """ """
+    with initialize_config_module(config_module="gtsfm.configs"):
+        # config is relative to the gtsfm module
+        cfg = compose(config_name="default_lund_door_set1_config.yaml")
+        scene_optimizer: SceneOptimizer = instantiate(cfg.SceneOptimizer)
 
-    sfm_result_graph = scene_optimizer.create_computation_graph(
-        len(loader),
-        loader.get_valid_pairs(),
-        loader.create_computation_graph_for_images(),
-        loader.create_computation_graph_for_intrinsics(),
-        use_intrinsics_in_verification=True,
-        gt_pose_graph=loader.create_computation_graph_for_poses(),
-    )
+        loader = FolderLoader(os.path.join(TEST_ROOT, "data", "set1_lund_door"), image_extension="JPG")
```

Comment:
should we have this from config too?

Reply:
I think we had talked about how the `SceneGenerator` would provide these paths to the `SceneOptimizer`, so I think they would go in a SceneGenerator config, not the SceneOptimizer config